File: 백준/Silver/14425. 문자열 집합/문자열 집합.py
import sys

input = sys.stdin.readline

n, m = list(map(int, input().split()))

check_list = []
dict = set()

count = 0

for i in range(n):
    word = input().strip()
    dict.add(word)

for j in range(m):
    word = input().strip()
    if word in dict:
        count += 1


print(count)

File: 백준/Silver/14425. 문자열 집합/문자열 집합.js
let input = require('fs').readFileSync('/dev/stdin').toString().trim().split('\n');
const [N, M] = input[0].split(' ').map(Number);
const NSet = new Set(input.slice(1, 1 + N));
const MList = input.slice(1 + N, 1 + N + M);

let count = MList.reduce((acc, word) => acc + (NSet.has(word) ? 1 : 0), 0);
console.log(count);

